qutebrowser/completion/completionwidget.py

```diff
@@ -255,3 +260,21 @@ def showEvent(self, e):
         if scrollbar is not None:
             scrollbar.setValue(scrollbar.minimum())
         super().showEvent(e)
+
+    def _open_completion_if_needed(self):
+        self._set_auto_open(not config.get('completion', 'auto-open'))
```

Comment:
I don't really get what's happening here - does this actually *open* something like the completion says? Why the negation?